README.md

```diff
@@ -1,183 +1,139 @@
 # llnode
 
-[![Build Status](https://travis-ci.org/nodejs/llnode.svg?branch=master)](https://travis-ci.org/nodejs/llnode)
+[![npm](https://img.shields.io/npm/v/llnode.svg?style=flat-square)](https://npmjs.org/package/llnode)
+[![Build Status](https://img.shields.io/travis/nodejs/llnode.svg?style=flat-square)](https://travis-ci.org/nodejs/llnode)
 
-Node.js v4.x-v8.x C++ plugin for the [LLDB](http://lldb.llvm.org) debugger.
+Node.js v4.x+ C++ plugin for the [LLDB](http://lldb.llvm.org) debugger.
 
 The llnode plugin adds the ability to inspect JavaScript stack frames, objects,
 source code and more to the standard C/C++ debugging facilities when working
 with Node.js processes or core dumps in LLDB.
 
+## Demo
+
+https://asciinema.org/a/29589
+
 ### Quick start
 
-Start an lldb session with the llnode plugin pre-loaded:
+Start an LLDB session with the llnode plugin automatically loaded:
 
 ```bash
 npm install -g llnode
-llnode node -c core
+llnode `which node` -c /path/to/core/dump
 ```
 
 - For more details on starting llnode see the [Usage](#usage) section.
 - To get started with the llnode commands see the [Commands](#commands) section.
 
-## Demo
-
-https://asciinema.org/a/29589
-
 ## Install Instructions
 
-To use llnode you need to have the LLDB debugger installed.
-
-- On OS X lldb is installed as part of Xcode. You will need Xcode both to build and run llnode.
-- On Linux install the lldb package using your distributions package manager.
+### Prerequisites: Install LLDB and its Library
 
-### Global install with npm
+To use llnode you need to have the LLDB debugger installed. The recommended
+version is LLDB 3.9 and above.
 
-```bash
-npm install -g llnode
-```
+- OS X/macOS
+  - You can install LLDB as part of [Xcode](https://developer.apple.com/xcode/).
```

Comment:
Maybe `use the LLDB that is part of` instead of `install LLDB as part of`?

The "you can" suggests it's optional or requires extra steps but I don't recall an option to disable it when I last installed xcode.

Reply:
@bnoordhuis I will update it to `You can install [Xcode](https://developer.apple.com/xcode/) and use the LLDB that comes with it` so the action that needs to be done is clearer.